src/fetch/fetchURL.js

```diff
@@ -17,22 +16,26 @@ const httpError = (err, res) => {
   const nerror = rewriteError(err)
   nerror.requestError = true
   nerror.code = res.code
-  nerror.status = res.status
+  nerror.status = res.statusCode
   nerror.headers = res.headers
-  nerror.body = res.text
```

Comment:
We still need to keep this behavior - the body should be included with the error. In the tests the body just happens to be the status code.

Reply:
You can update the test bodies to make sure this doesn't get overlooked again ^